webcompat/static/js/lib/bugform.js

```diff
@@ -292,11 +297,13 @@ function BugForm() {
 
   /* Check if Browser and OS are empty or not, only
      so we can set them to valid (there is no invalid state) */
-  this.checkOptionalNonEmpty = function() {
+  this.checkOptionalNonEmpty = function(types) {
     _.forEach(
-      [this.browserField, this.osField],
-      _.bind(function(input) {
+      [...types],
```

Comment:
We can't use es6 rest because it will break the reporting form for IE users, and other older browsers. :(

Reply:
Sorry, I shouldn't switch between projects. 😅
Will send a fix.